src/main/cli.ts

```diff
@@ -0,0 +1,98 @@
+import { KANGAROO_CONFIG } from './const';
+import { breakingAppVersion } from './filesystem';
+import { app } from 'electron';
+
+export interface CliOpts {
+  profile?: string;
+  networkSeed?: string | undefined;
+  holochainPath?: string | undefined;
+  lairPath?: string | undefined;
+  holochainRustLog?: string | undefined;
+  holochainWasmLog?: string | undefined;
+  lairRustLog?: string | undefined;
+  bootstrapUrl?: string;
+  signalingUrl?: string;
+  iceUrls?: string;
+  printHolochainLogs?: boolean;
+}
+
+export interface RunOptions {
+  profile: string | undefined;
+  networkSeed: string;
+  bootstrapUrl: string | undefined;
+  signalingUrl: string | undefined;
+  iceUrls: string[] | undefined;
+  customHolochainBinary: string | undefined;
```

Comment:
Is this different to the `holochainPath` parameter above? Should the two be named the same?

Reply:
My reasoning here was to name it in a way that it is less likely to be confused with the directory where the conductor database is being stored (and which currently goes by the name `conductorDir`). So it could also be named the same...

Reply:
I'd prefer the names match if they carry the same value. If they both have to be renamed to some common name then I'd be happy with that.

This isn't super important to me though, I will leave it up to you